amoro-format-iceberg/src/main/java/org/apache/amoro/formats/mixed/MixedSnapshot.java

```diff
@@ -42,10 +42,15 @@ public long watermark() {
 
   @Override
   public long commitTime() {
-    Long changCommit =
-        Optional.ofNullable(changeSnapshot).map(Snapshot::timestampMillis).orElse(-1L);
-    Long baseCommit = Optional.ofNullable(baseSnapshot).map(Snapshot::timestampMillis).orElse(-1L);
-    return Longs.max(changCommit, baseCommit);
+    return Longs.max(getChangeCommit(), getBaseCommit());
+  }
+
+  public long getChangeCommit() {
```

Comment:
Why is this function declared as public?

If this method is only meant for internal usage, consider using private or protected to better encapsulate the implementation details.